examples/download_and_train.py

```diff
@@ -0,0 +1,101 @@
+import t4
+import os
+import tqdm
+import json
+import pandas as pd
+
+from fnet.cli.init import save_default_train_options
+
+###################################################
+# Download the 3D multi-channel tiffs via Quilt/T4
+###################################################
+
+gpu_id = 0
+
+n_images_to_download = 40
+train_fraction = 0.75
+
+image_save_dir = '{}/images/'.format(os.getcwd())
+model_save_dir = '{}/model/'.format(os.getcwd())
+prefs_save_path = '{}/prefs.json'.format(os.getcwd())
+
+data_save_path_train =  '{}/image_list_train.csv'.format(image_save_dir)
+data_save_path_test =  '{}/image_list_test.csv'.format(image_save_dir)
+
+if not os.path.exists(image_save_dir):
+    os.makedirs(image_save_dir)
+
+
+aics_pipeline = t4.Package.browse(
+    "aics/pipeline",
+    registry="s3://quilt-aics"
+)
+
+len(aics_pipeline['fov'])
+
+image_ids = [k for k in aics_pipeline['fov']][0:n_images_to_download]
+
+metadata = {}
+for image_id in image_ids:
+    metadata[image_id] = aics_pipeline['fov'][image_id].get_meta()
+
+image_save_paths = ['{}/{}'.format(image_save_dir, image_id) for image_id in image_ids]    
```

Comment:
With `pathlib.Path`s this becomes:
`image_save_paths = [image_save_dir / image_id for image_id in image_ids`

But again, metadata storage and remaking the files list wont be necessary once the `t4` fetch update PR goes through.